openapi.yml

```diff
@@ -567,6 +567,10 @@ components:
             - 'citation-only'
             - 'dark'
           default: 'dark'
+        controlledDigitalLending:
```

Comment:
where does the camel casing happen?   trying to understand how this maps to the work in dor-services-app

Reply:
I just typed it in this way.